.github/workflows/release.yml

```
@@ -12,35 +12,47 @@ on:
         default: "auto"
         required: true
 
+concurrency:
+  group: ${{workflow}}-${{github.ref}}
```

actionlint (1.7.12)

[error] undefined variable "workflow". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)